client/static/index.scss

```diff
@@ -0,0 +1,157 @@
+$coolBlue: #032442;
```

Comment:
Just as the hound bot has warned, your variable names in sass/scss should be in this format `$cool-blue` or `$dark-red`.

Reply:
okay. Noted